wild-swimming-app/src/pages/home/Homepage.tsx

```diff
@@ -1,10 +1,15 @@
 import Header from "../../components/Header/Header";
+import Map from "../../feature/map/Map";
 
 const Homepage = () => {
   return (
     <section>
+
       <Header />
       <p>Homepage</p>
+      <Map/>
```

Comment:
I can see there is a conflict with this merge, was Map left in for your local visibility? 
I am happy for this to be added in ahead of project presentation, but just would want that Homepage mmerges aren't removing anything (which it doesn't look like!)